src/ClimaLSM.jl

```diff
@@ -153,17 +156,20 @@ function make_exp_tendency(land::AbstractLandModel)
     compute_exp_tendency_list =
         map(x -> make_compute_exp_tendency(getproperty(land, x)), components)
     update_aux! = make_update_aux(land)
+    update_boundary_fluxes! = make_update_boundary_fluxes(land)
     function exp_tendency!(dY, Y, p, t)
         update_aux!(p, Y, t)
+        update_boundary_fluxes!(p, Y, t)
         for f! in compute_exp_tendency_list
             f!(dY, Y, p, t)
         end
     end
     return exp_tendency!
 end
 
+# this needs thought
```

Comment:
it may be OK, but each `set_initial_aux_state` for each component also updates that components boundary vars. so we call those 2x. but not sure how to get around it.